index.d.ts

```diff
@@ -0,0 +1,25 @@
+import { FastifyPlugin } from "fastify";
+import { LevelUp } from "levelup";
+
+export interface LevelDBOptions {
+  name: string,
+  path: string
+}
+
+type LevelDBInstances {
+  [key: string]: LevelUp
+}
+
+// Most importantly, use declaration merging to add the custom property to the Fastify type system
+declare module 'fastify' {
+  interface FastifyInstance {
+    level: LevelDBInstances
+  }
+}
+
+// fastify-plugin automatically adds named export, so be sure to add also this type
+// the variable name is derived from `options.name` property if `module.exports.myPlugin` is missing
+export const levelDBPlugin: FastifyPlugin<LevelDBOptions>;
```

Comment:
According to https://github.com/fastify/fastify-plugin/blob/master/plugin.js#L78-L81
It should be `fastifyLeveldb`.
```suggestion
export const fastifyLeveldb: FastifyPlugin<LevelDBOptions>;
```